Example/Emission/AppDelegate.m

```diff
@@ -251,7 +252,7 @@ - (UIViewController *)viewControllerForRoute:(NSString *)route;
 - (void)dismissModalViewController;
 {
   UINavigationController *navigationController = (UINavigationController *)self.window.rootViewController;
-  [navigationController.visibleViewController.navigationController dismissViewControllerAnimated:YES completion:nil];
+  [navigationController dismissViewControllerAnimated:YES completion:nil];
```

Comment:
The above code was causing a crash. It looks like it was never/seldom called, so it's likely been a but in the Emission test app. Generally speaking, UIKit handles walking up the view controller hierarchy to dismiss view controllers for us automatically.

Reply:
Pretty sure this is to support having storybooks with navigation elements inside them

Reply:
Okay, I'll take a look at see if this breaks things.

Reply:
So this code was [added two years ago](https://github.com/artsy/emission/commit/ce2a6f376ff4cbb13ebd5fcd100fcc3554fd0179) strictly as a mechanism for handling `presentModalViewController` from JS. I've looked at other cases where `presentModalViewController` is used and this still works. I think we're 👍 on this change.

This is also only in the Emission Example app, not the pod, so anything we inadvertently break is only visible to people testing the Emission beta app.

Reply:
:+1: